libs/source_post_processors/objects_filtering.py

```diff
@@ -80,11 +91,70 @@ def non_max_suppression_fast(object_list, overlapThresh):
         updated_object_list = [j for i, j in enumerate(object_list) if i in pick]
         return updated_object_list
 
+    @staticmethod
+    def is_inside_roi(detected_object, roi_contour):
+        """
+        An object is inside the RoI if its middle bottom point lies inside it.
+        params:
+            detected_object: a dictionary, that has attributes of a detected object such as "id",
+            "centroid" (a tuple of the normalized centroid coordinates (cx,cy,w,h) of the box),
+            "bbox" (a tuple of the normalized (xmin,ymin,xmax,ymax) coordinate of the box),
+            "centroidReal" (a tuple of the centroid coordinates (cx,cy,w,h) of the box) and
+            "bboxReal" (a tuple of the (xmin,ymin,xmax,ymax) coordinate of the box)
+
+            roi_contour: An array of duples that compose the contour of the RoI
+        returns:
+        True of False: Depending if the objects coodinates are inside the RoI
+        """
+        corners = detected_object["bboxReal"]
+        x1, x2 = int(corners[0]), int(corners[2])
+        y1, y2 = int(corners[1]), int(corners[3])
+        if cv.pointPolygonTest(roi_contour, (x1 + (x2-x1), y2), False) >= 0:
```

Comment:
This equals to (x2, y2). There is a `/ 2` missing

Reply:
Well spotted!